### PR TITLE
Update general-remarks.md

### PR DESCRIPTION
**Pull request by patrikmiksovsky95**

### CODE REVIEW

File: general-remarks.md [hunk 1]
```diff
@@ -121,6 +125,12 @@ Or
 | `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
 | `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
 | `8` | Invalid authorization |
+| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |
+| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. | 
+| `11` | Rate category error - e.g. category was removed from the rate plan in channel manager extranet. |
+| `12` | Availability configuration error - e.g., availability updates blocked from the channel manager extranet. |
+| `13` | Prices configuration error - e.g., price updates blocked from the channel manager extranet. |
+| `14` | Restrictions configuration error - e.g. operation is disabled in channel manager extranet. |
```
Comment (Ksenia-Galtsova) [inline general-remarks.md:133]: Let's follow the same format everywhere, so it won't cause any confusion.

e.g., restriction updates blocked from the channel manager extranet.

File: general-remarks.md [hunk 2]
```diff
@@ -121,6 +125,12 @@ Or
 | `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
 | `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
 | `8` | Invalid authorization |
+| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |
+| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. |
```
Comment (Ksenia-Galtsova) [inline general-remarks.md:129]: missing one word: space category XX availability updates cannot be accepted due to the settings in the channel manager extranet.

File: general-remarks.md [hunk 3]
```diff
@@ -121,6 +125,12 @@ Or
 | `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
 | `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
 | `8` | Invalid authorization |
+| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |
+| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. | 
+| `11` | Rate category error - e.g. category was removed from the rate plan in channel manager extranet. |
```
Comment (Ksenia-Galtsova) [inline general-remarks.md:130]: e.g., such rate plan-space category combination is not created in the channel manager extranet.

File: general-remarks.md (hunk 2 above)
Comment (patrikmiksovsky95) [inline general-remarks.md:129]:
```suggestion
| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager extranet. | 
```

File: general-remarks.md [hunk 4]
```diff
@@ -121,6 +125,12 @@ Or
 | `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
 | `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
 | `8` | Invalid authorization |
+| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |
+| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager extranet. | 
+| `11` | Rate category error - e.g. category was removed from the rate plan in channel manager extranet. |
+| `12` | Availability configuration error - e.g., availability updates blocked from the channel manager extranet. |
+| `13` | Prices configuration error - e.g., price updates blocked from the channel manager extranet. |
+| `14` | Restrictions configuration error - e.g. operation is disabled in channel manager extranet. |
```
Comment (patrikmiksovsky95) [inline general-remarks.md:133]:
```suggestion
| `14` | Restrictions configuration error - e.g., restriction updates blocked from the channel manager extranet. |
```

**Review by Ksenia-Galtsova — APPROVED** [review]

File: general-remarks.md [hunk 5]
```diff
@@ -100,14 +100,18 @@ Or
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `success` | `bool` | required | `true` or `false` determining result of operation. |
-| `error` | [`Error`](general-remarks.md#error) | optional | In case of `"success": false`, this property holds information of the error that happened. |
+| ~~`error`~~ | ~~[`Error`](general-remarks.md#error)~~ | ~~optional~~ | ~~In case of `"success": false`, this property holds information of the error that happened.~~ |
```
Comment (siroky) [inline general-remarks.md:103]: Do you have followup cleanup tasks for all such places?

Comment (patrikmiksovsky95) [reply general-remarks.md:103]: From the far past we dont now i will connect it with the deprecation issue.

Comment (siroky) [reply general-remarks.md:103]: The you should figure it out and remove all the things that are currently crossed out and have been like this for long time. You should also remove it from the API completely.

**Pull request merged**